frontend/app/src/components/StreamlitDialog/ThemeCreatorDialog.test.tsx

```diff
@@ -218,25 +215,11 @@ describe("Opened ThemeCreatorDialog", () => {
     )
   })
 
-  it("should have font dropdown populated", () => {
```

Comment:
This will be checked in the test above if we are clicking the font drop down and selecting a font.